contracts/reverseRegistrar/IL2ReverseRegistrar.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity >=0.8.4;
+
+interface IL2ReverseRegistrar {
+    function setName(string memory name) external returns (bytes32);
+
+    function setNameForAddr(
+        address addr,
+        string memory name
+    ) external returns (bytes32);
+
+    function setNameForAddrWithSignature(
+        address addr,
+        string memory name,
+        uint256 inceptionDate,
+        bytes memory signature
+    ) external returns (bytes32);
+
+    function setNameForAddrWithSignatureAndOwnable(
+        address contractAddr,
```

Comment:
I think it reads better if the argument order is in line with the function name then all the rest (name, adddr, signature, owner, contractAddress, inceptionDate...). Same for other functions